streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -1085,77 +1086,72 @@ public void shouldAllowToQueryAfterThreadDied() throws Exception {
         final StreamsBuilder builder = new StreamsBuilder();
         final KStream<String, String> input = builder.stream(streamOne);
         input
-            .groupByKey()
-            .reduce((value1, value2) -> {
-                if (value1.length() > 1) {
-                    if (beforeFailure.compareAndSet(true, false)) {
-                        throw new RuntimeException("Injected test exception");
+                .groupByKey()
```

Comment:
nit: seems this is just reformatting changes? (Make the diff hard to read -- please avoid, so we can focus on actual changes)

Reply:
I understand, though(interesting), that I didn't apply any reformatting. Let me check

Reply:
I bet it's auto-reformatting enabled in your IDE -- you did open the file an modify it, and the IDE reformatted w/o you noticing it. -- I usually recommend to disable auto-formatting setting in an IDE.